apps/web/components/BackgroundImage/BackgroundImage.tsx

```diff
@@ -59,7 +65,22 @@ export const BackgroundImage = ({
       }
     : {}
 
-  const imageLoaded = useImageLoader(src)
+  useLayoutEffect(() => {
+    if (backgroundImageRef?.current && 'IntersectionObserver' in window) {
+      const lazyBackgroundObserver = new IntersectionObserver((entries) => {
```

Comment:
what is the minimum browser requirement? is this already polyfilled?

Reply:
I found a similar tool already in the package.json (react-use, https://github.com/streamich/react-use/blob/master/docs/useIntersection.md).